docs/reference/snapshot-restore/apis/restore-snapshot-api.asciidoc

```diff
@@ -118,14 +115,17 @@ operation initializes. Defaults to `false`.
 ==== {api-request-body-title}
 
 `ignore_unavailable`::
-(Optional, Boolean)
-If `false`, the request returns an error for any data stream or index that is missing or closed. Defaults to `false`.
-+
-If `true`, the request ignores data streams and indices in `indices` that are missing or closed.
+(Optional, Boolean) If `true`, the request ignores any index or data stream in
+`indices` that's missing from the snapshot. If `false`, the request returns an
+error for any missing index or data stream. Defaults to `false`.
 
 `ignore_index_settings`::
-(Optional, string)
-A comma-separated list of index settings that should not be restored from a snapshot.
+(Optional, array of string) Index settings to not restore from the snapshot. You
```

Comment:
This can be either an array of strings or a single string containing the setting names separated by commas:

https://github.com/elastic/elasticsearch/blob/a5d2251064d3457dc253503ba9ce75dd57fa6f4e/server/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/restore/RestoreSnapshotRequest.java#L526-L530

Reply:
Good catch. Thanks!